configs/pontoon.toml

```diff
@@ -164,8 +164,34 @@ locales = [
     reference = "en/firefox/set-as-default/**/*.ftl"
     l10n = "{locale}/firefox/set-as-default/**/*.ftl"
 [[paths]]
-    reference = "en/firefox/welcome/**/*.ftl"
-    l10n = "{locale}/firefox/welcome/**/*.ftl"
+    reference = "en/firefox/welcome/page1.ftl"
+    l10n = "{locale}/firefox/welcome/page1.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page2.ftl"
+    l10n = "{locale}/firefox/welcome/page2.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page3.ftl"
+    l10n = "{locale}/firefox/welcome/page3.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/page4.ftl"
+    l10n = "{locale}/firefox/welcome/page4.ftl"
+[[paths]]
```

Comment:
Should we add page5?

Reply:
Page 5 is hard coded and does not use an .ftl

Reply:
It's in Pontoon: https://pontoon.mozilla.org/de/mozillaorg/en/firefox/welcome/page5.ftl/?string=212989